serialization/src/main/java/org/nervos/ckb/type/Bytes.java

```diff
@@ -0,0 +1,42 @@
+package org.nervos.ckb.type;
+
+import org.nervos.ckb.utils.Numeric;
+
+/** Copyright © 2019 Nervos Foundation. All rights reserved. */
+public class Bytes implements Type<byte[]> {
+
+  private byte[] value;
+
+  public Bytes(byte[] value) {
+    this.value = value;
+  }
+
+  public Bytes(String value) {
+    this.value = Numeric.hexStringToByteArray(value);
+  }
+
+  public static Bytes fromBytes(byte[] bytes) {
+    byte[] dest = new byte[bytes.length - UInt32.BYTE_SIZE];
+    System.arraycopy(bytes, UInt32.BYTE_SIZE, dest, 0, dest.length);
+    return new Bytes(dest);
+  }
+
+  @Override
+  public byte[] toBytes() {
+    byte[] lens = new UInt32(value.length).toBytes();
```

Comment:
It will be better to unite with Table.
`System.arraycopy(new UInt32(length).toBytes(), 0, dest, 0, UInt32.BYTE_SIZE)`